### ISSUE TITLE
[1.12][Request] Tools

### ISSUE THREAD

**Issue by NillerMedDild** [RootsTeam/Embers#232]
Hey!

I was wondering if you wouldn't mind making a config option, to turn off copper/silver/tin/lead tools? :)

**Comment by elucent** [comment]
Why?

**Comment by NillerMedDild** [comment]
Because they're also added by Thermal Foundation, meaning they're uncraftable anyhow.

**Comment by bookerthegeek** [comment]
Most likely to condense the ampint of tools in game. I know IC2 adds bronze tools, amd cofh adds them for most metals. This causes either some tools that can not be crafted or the need to minetweaker one set of tools out of the game.

If you decide to do this, a true/false for each netal type would be appreciated.

**Comment by elucent** [comment]
Yeah, as far as convenience goes, this is a pretty reasonable feature and it shouldn't be too hard to add.
